pkg/plugins/gateway/algorithms/least_busy_time.go

```diff
@@ -26,9 +26,7 @@ import (
 	klog "k8s.io/klog/v2"
 )
 
-var (
-	RouterLeastBusyTime types.RoutingAlgorithm = "least-busy-time"
-)
+const RouterLeastBusyTime types.RoutingAlgorithm = "least-busy-time"
```

Comment:
for such changes not directly related to this change, let's cut separate PRs in future